packages/dom/test/serialize-dom.test.js

```diff
@@ -34,6 +34,25 @@ describe('serializeDOM', () => {
     expect(serializeDOM().html).toMatch('<!DOCTYPE html>');
   });
 
+  it('does not trigger DOM events on clone', () => {
+    class CallbackTestElement extends window.HTMLElement {
+      connectedCallback() {
+        const wrapper = document.createElement('h2');
+        wrapper.className = 'callback';
+        wrapper.innerText = 'Test';
+        this.appendChild(wrapper);
+      }
+    }
+
+    if (!window.customElements.get('cllback-test')) {
```

Comment:
```suggestion
    if (!window.customElements.get('callback-test')) {
```

Reply:
does this test fail without the above ^ code changes?

Reply:
no its just a sanity check

Reply:
I meant, the code changes of `v1.19.2-alpha.0`